apps/web/app/[lang]/(use-page-wrapper)/(main-nav)/availability/page.tsx

```diff
@@ -7,16 +9,13 @@ import { notFound } from "next/navigation";
 // import { OrganizationRepository } from "@calcom/lib/server/repository/organization";
 import AvailabilityPage, { AvailabilityCTA } from "~/availability/availability-view";
 
-import { ShellMainAppDir } from "../ShellMainAppDir";
+export const generateMetadata = async ({ params }: PageProps) => {
+  const t = await getTranslate(params.lang as string);
```

Comment:
do we want to update PageProps, so that we don't have to `as string`

Reply:
Addressed!